eng/Version.Details.xml

```diff
@@ -1,16 +1,16 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.4-servicing.20315.1">
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.4-servicing.20366.2">
```

Comment:
WHy are these stuck at 3.1.4?

Reply:
i fixed these in https://github.com/dotnet/coreclr/pull/28075
I was missing the place where we update the transport packages